src/content/contribute/writers-guide.md

```diff
@@ -159,3 +160,47 @@ Please do not assume things are simple. Avoid words like 'just', 'simply'.
 - Simply run command...
 + Run the `command-name` command...
 ```
+
+### Configuration defaults and types
+
+Always provide types and defaults to all of the documentation options in order to keep documentation accessible and well-written. We are adding types and defaults after entitling the documented option:
+
+__configuration.example.option__
+
+`string = 'none'`
+
+Where `= 'none'` means that default value is `'none'` for the given option.
+
+`string = 'none': 'none' | 'development' | 'production'`
+
+Where `: 'none' | 'development' | 'production'` enumerates the possible type values, in this case, three strings are acceptable: `'none' | 'development' | 'production'`.
+
+Use space between types to list all available types for the given option:
+
+`string = 'none': 'none' | 'development' | 'production'` `boolean`
+
+To mark an array, use square brackets:
+
+`string` `[string]`
+
+To mark a function, also list arguments when they are available:
+
+`function (compilation, module, path) => boolean`
+
+Where `(compilation, module, path)` lists the arguments that the provided function will receive and `=> boolean` means that return value of the function must be a `boolean`.
+
+To mark a Plugin as an available option value type, use camel case title of the Plugin:
+
+`TerserPlugin` `[TerserPlugin]`
+
+Which means that option exptects one or few `TerserPlugin` instances.
+
+To mark a number, use `number`:
+
+`number = 15: 5, 15, 30`
+
+To mark an object, use `object`
+
+`object = { prop1 string = true, prop2 boolean = false, prop3 function (module) => string}`
```

Comment:
If I understand this correctly, `prop1` is a `string` with the default value `true`. Shouldn't the default value be a `string` and not a `boolean`?

Reply:
yup good catch!
```suggestion
`object = { prop1 string = 'none', prop2 boolean = false, prop3 function (module) => string}`
```